packages/modals/src/index.js

```diff
@@ -15,3 +15,4 @@ export { default as Footer } from './views/Footer';
 export { default as FooterItem } from './views/FooterItem';
 export { default as Header } from './views/Header';
 export { default as ModalView } from './views/ModalView';
+export { default as useModalContext, ModalContext } from './utils/useModalContext';
```

Comment:
Should I export this? Reason being is if you want to use the view components outside of the `<Modal>` component you need to wrap it in context that passes down the prop getters. See the `ModalView.example.md` as an example of that.